StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlCommentHelper.cs

```diff
@@ -268,20 +267,101 @@ internal static string GetText(XmlTextSyntax textElement, bool normalizeWhitespa
                 return null;
             }
 
-            StringBuilder stringBuilder = StringBuilderPool.Allocate();
+            bool lastWhitespace = false;
+
+            string single = string.Empty;
+
+            StringBuilder stringBuilder = null;
 
             foreach (var item in textElement.TextTokens)
             {
-                stringBuilder.Append(item);
+                if (single.Length == 0)
+                {
+                    single = item.ToString();
+                }
+                else
+                {
+                    if (stringBuilder == null)
+                    {
+                        stringBuilder = StringBuilderPool.Allocate();
+                        stringBuilder.AppendNormalize(single, normalizeWhitespace, ref lastWhitespace);
+                    }
+
+                    stringBuilder.AppendNormalize(item.ToString(), normalizeWhitespace, ref lastWhitespace);
```

Comment:
❔ Is it common for this line to be hit when `lastWhitespace` is true _and_ `item` is pure whitespace?